.go-version

```diff
@@ -0,0 +1 @@
+1.16.6
```

Comment:
`go.mod` has `1.17` in it, can we update it?

Reply:
Indeed, you can amend the changes here if needed :)